Ports/open-adventure/patches/ReadMe.md

```diff
@@ -0,0 +1,13 @@
+# Patches for open-adventure on SerenityOS
+
+## `0001-Use-libeditline-instead-of-libedit.patch`
+
+Use [`editline`](../../editline/) instead of `libedit`.
+
+## `0002-Change-headers-for-libeditline.patch`
+
+Change headers because we are now using [`editline`](../../editline/).
+
+## `0003-Rearrange-headers-because-editline.h-uses-FILE.patch`
+
+Move `stdio.h` before `editline.h` because it uses `FILE`.
```

Comment:
Please merge those three patches, they all are related to using libeditline instead of libedit.

Reply:
Done.